crowd_anki/github/github_importer.py

```diff
@@ -32,24 +28,15 @@ def import_from_github(self):
         repo, ok = QInputDialog.getText(None, 'Enter GitHub repository',
                                         'Path:', text='<name>/<repository>')
         if repo and ok:
-            self.download_and_import(repo)
+            self.download_and_import_git(repo)
```

Comment:
why rename?

Reply:
Good point. It was because "download and import" before used to mean "download zip and import". So now it could be renamed to "clone_and_import", perhaps?